playwright/tests/e2e/user.spec.ts

```diff
@@ -0,0 +1 @@
+// TODO: Realize test scanarios for user from MIRO
```

Comment:
realize test scenarios?

Reply:
Markdown for future code. Should I remove two empty files?

Reply:
Remove empty files, or add basic scenario